frontend/views/pages/PayGroup.vue

```diff
@@ -20,65 +27,205 @@ page(
-    i18n.has-text-1(tag='p') Once other group members add their income details, Group Income will re-distribute wealth amongst everyone.
-
-  section.card(v-else)
-    i18n.is-title-3(
-      tag='h3'
-      :args='{ month: paymentStatus.month }'
-    ) {month} contributions
-    i18n.has-text-1(
-      tag='p'
-      :args='{ amount: paymentTotal }'
-    ) {amount} in total
-
-    ul.c-payments
-      li.c-payments-item(
-        v-for='(payment, index) in paymentsDistribution'
+  i18n.p-descritpion.has-text-small.has-text-1(
+    tag='p'
+    data-test='openIncomeDetailsModal'
+    @click='openModal("IncomeDetails")'
+    :args='{ receive_or_send: needsIncome ? L("receiving") : L("sending"), r1: `<button class="link js-btnInvite">`, r2: "</button>"}'
+  ) You are currently {receive_or_send} mincome. You can change this at any time by updating your {r1}income details{r2}.
+
+  section.card
+    nav.tabs(
+      v-if='!needsIncome'
+      aria-label='navigation'
+      @click='open = false'
+    )
+      a.tabs-link(
+        v-for='(link, index) in tabItems'
         :key='index'
+        :class='{ "tabs-link-active": activeTab === link.url}'
+        :data-test='`link-${link}`'
+        @click='activeTab = link.url'
       )
-        .c-info
-          avatar-user.c-avatar(:username='payment.to' size='sm')
-          div(role='alert')
-            i18n(
-              tag='p'
-              :args='{ \
-                total: `<b>${payment.amountFormatted}</b>`, \
-                user: `<b>${payment.to}</b>` \
-              }'
-            ) {total} to {user}
-            span(
-              v-if='payment.hash'
-              :class='payment.paymentClass'
-            ) {{ payment.paymentStatusText }}
-
-        .buttons.is-start.c-ctas
-          router-link.button.is-small.is-outlined(
-            v-if='payment.data.status === statusType.PAYMENT_NOT_RECEIVED'
-            to='messages/liliabt'
-          ) {{ L('Send Message...') }}
-          i18n.button.is-small.c-ctas-send(
-            tag='button'
-            key='send'
-            v-if='showMarkAsPaid(payment)'
-            @click='markAsPayed(payment.to, payment)'
-          ) Mark as sent
-          i18n.button.is-small.is-outlined(
-            tag='button'
-            key='cancel'
-            v-if='payment.data.status === statusType.PAYMENT_PENDING'
-            @click='cancelPayment(payment.to, payment.hash)'
-          ) Cancel
-
-  .c-footer
-    i18n.button.is-small.is-outlined(
-      tag='button'
-      @click='seeHistory'
-    ) See past contributions
+        | {{ link.title }}
+        span.tabs-notification(v-if='link.notification') {{ link.notification }}
+
+    form.c-search-form(
+      v-if='needsIncome || activeTab !== "todo"'
+      @submit.prevent='submit'
+    )
+      label.field
+        i18n.sr-only Search payments
+        .input-combo
+          .is-icon(:aria-label='L("Search")')
```

Comment:
We already have the description on line 58 "Search payments", I think there's no need to add it here too. It would be redundant.